tests/neg/i14823.check

```diff
@@ -2,5 +2,5 @@
 8 |val baz = summon[Mirror.Of[SubA[Int] | SubB[Int]]] // error
   |                                                  ^
   |No given instance of type deriving.Mirror.Of[SubA[Int] | SubB[Int]] was found for parameter x of method summon in object Predef. Failed to synthesize an instance of type deriving.Mirror.Of[SubA[Int] | SubB[Int]]: 
-  |	* class Cov is not a generic sum because it is not a sealed class
   |	* class Cov is not a generic product 
```

Comment:
this is another interesting one, to the user seeing this error, it might not be obvious why the error mentions `Cov`, and not `SubA` or `SubB`, I think this can wait for a new PR but perhaps we should explain why the union collapses to its common superclass.